consul-lambda-registrator/proxy/proxy_test.go

```diff
@@ -0,0 +1,298 @@
+package proxy_test
```

Comment:
This is new. It tests that the proxy can handle plaintext and TLS for L4 TCP and that HTTP is also supported